backend/src/main/java/storage/PostgresSQLConnector.java

```diff
@@ -0,0 +1,109 @@
+package storage;
+
+
+
+import model.Customer;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+public class PostgresSQLConnector implements storage.Persistance {
+
+    private static final String CONNECTION_URL = "jdbc:postgresql://db:5432/svereadb";
+    private static final String USERNAME = "admin";
+    private static final String PWD = "1234";
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') 'PWD' detected in this expression, review this potentially hardcoded credential. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2068)